### ISSUE TITLE
Import all ndimage functions into a single namespace

### ISSUE THREAD

**Issue by GenevieveBuckley** [dask/dask-image#198]
To allow for better dispatching & interoperability with other libraries, dask-image should make sure that all the `ndimage` functions are imported into a single namespace.

We need to:
1. Decide what that namespace should be: `dask_image.scipy` or `dask_image.scipy.ndimage` or `dask_image`? @jakirkham what do you think?
2. Make sure all the functions are imported into that namespace. Once the decision for (1) is made, this is a good first issue.

Discussion:
- https://github.com/scipy/scipy/issues/10204#issuecomment-788529136
- https://github.com/scipy/scipy/issues/10204#issuecomment-801848146

**Comment by jakirkham** [comment]
Yeah I think this makes sense and opens the door to considering other things like scikit-image APIs for functions (when they might be needed here as opposed to be adapted to handle other arrays generally upstream)

**Comment by GenevieveBuckley** [comment]
Ok, sounds like we want to have the namespace `dask_image.scipy.ndimage`

**Comment by grlee77** [comment]
A related issue is differences in parameter names. For example, the functions in `ndmeasure` use `label_image` instead of SciPy's `labels` which causes incompatibility if the arguments are not passed positionally. Also, the first argument has the unfortunate name `input` rather than `image` in SciPy.

One option rather than changing the names (the ones used here do seem better), is to just have a thin wrapper using the SciPy names within the `dask_image.scipy.ndimage` namespace.

**Comment by jakirkham** [comment]
Interesting. That didn't use to be the case. Appears to have changed in PR ( https://github.com/dask/dask-image/pull/126 ). Maybe it can be reverted?

Right that was raised upstream in issue ( https://github.com/scipy/scipy/issues/10592 ). I don't think we want to go back to `input` here as that overlaps with a builtin. It would be preferable if whatever API we share doesn't use `input`.
